recipes/monitoring_collectd.rb

```diff
@@ -23,6 +23,33 @@
 end
 
 include_recipe 'yum-epel' if node['platform_family'] == 'rhel'
+include_recipe 'selinux_policy::install' if node['platform_family'] == 'rhel'
+
+selinux_policy_module 'MY_collectd' do
```

Comment:
instead of using MY_, I suggest using RS_ or rs_ or better yet rightscale_ to show that  rightscale created the policy.